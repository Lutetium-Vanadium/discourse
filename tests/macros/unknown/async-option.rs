fn main() {
    let q = inquisition::questions![input { async name: 0 }];
}
