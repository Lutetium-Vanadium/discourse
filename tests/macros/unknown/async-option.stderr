error: unknown question option `name_async`
 --> $DIR/async-option.rs:2:51
  |
2 |     let q = inquisition::questions![input { async name: 0 }];
  |                                                   ^^^^
